VERSIONING.md

```diff
@@ -22,7 +22,3 @@ MAJOR version when you make incompatible API changes,
 MINOR version when you add functionality in a backwards-compatible manner, and
 PATCH version when you make backwards-compatible bug fixes.
 Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
-
-## Standardized tooling
-
-each project should implement `bin/bump-version.sh {newVer}`
```

Comment:
rel #13, thinking of just removing this since its really based on tooling, for example semantic release will auto generate a major bump based on if it has `BREAKING` changes

Reply:
yea, i'd concur on a remove here